.github/workflows/linters.yml

```diff
@@ -0,0 +1,16 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  nodechecker:
+    name: node_modules checker
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check node_modules existence
+        run: |
+          if [ -d "node_modules/" ]; then echo -e "\e[1;31mThe node_modules/ folder was pushed to the repo. Please remove it from the GitHub repository and try again."; echo -e "\e[1;32mYou can set up a .gitignore file with this folder included on it to prevent this from happening in the future." && exit 1; fi
```

Comment:
- [ ] This linters.yml is not the correct one, because it has no rubocop.  Kindly add the correct yml file so the repo can check for linter errors.
- [ ] Your app has many linter errors (way too many to list here).  Kindly correct them.  Most of them can be autocorrected.

That way, your code will comply with highest standards. 💪